_test_unstructured_client/integration/test_decorators.py

```diff
@@ -1,43 +1,43 @@
 import os
+
 import pytest
 import requests
 from deepdiff import DeepDiff
-
 from unstructured_client import UnstructuredClient
 from unstructured_client.models import shared
 from unstructured_client.models.errors import HTTPValidationError
 
 FAKE_KEY = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
 
 
-@pytest.mark.parametrize("call_threads", [1, 2, 5])
+@pytest.mark.parametrize("concurrency_level", [1, 2, 5])
 @pytest.mark.parametrize(
-    "filename, expected_ok",
+    ("filename", "expected_ok", "strategy"),
     [
-        ("_sample_docs/list-item-example-1.pdf", True),  # 1 page
-        ("_sample_docs/layout-parser-paper-fast.pdf", True),  # 2 pages
-        ("_sample_docs/layout-parser-paper.pdf", True),  # 16 pages
-        ("_sample_docs/fake.doc", True),
+        ("_sample_docs/list-item-example-1.pdf", True, "fast"),  # 1 page
+        ("_sample_docs/layout-parser-paper-fast.pdf", True, "fast"),  # 2 pages
+        # NOTE(mike): using "fast" strategy fails on this file for unknown reasons
+        ("_sample_docs/layout-parser-paper.pdf", True, "hi_res"),  # 16 pages
+        ("_sample_docs/fake.doc", True, "fast"),
```

Comment:
Couldn't find a good, representative PDF example. On https://github.com/Unstructured-IO/unstructured/tree/main/example-docs, we only have tiny PDFs (which won't be helpful when testing the splitting logic), except for this one with 619 pages (https://github.com/Unstructured-IO/unstructured/blob/main/example-docs/DA-619p.pdf), which would slow down CI by about 9 minutes. Are there any better candidates? Please suggest them if so.

Reply:
Hmm I think the existing layout-parser-paper.pdf with 16 pages is good enough for CI. If the logic works with that much, then it should work with 2x more, 4x more etc. because it's not 1..5 edge case. But what I meant in the comment, was to add other popular filetypes like jpg png txt csv, to test that the non pdf detection works fine and doesn't throw unexpected errors.